README.md

```diff
@@ -431,6 +432,22 @@ Please see [Working with Panes](#working-with-panes) how to configure the defaul
 
 ## Usage
 
+### Shortcut cheatsheet
+
+* [Create a new note](#creating-a-new-note) <kbd>shift</kbd> + <kbd>enter</kbd>
+* [New note from text](#creating-a-new-note-and-link-from-selected-text) Select text, then <kbd>shift</kbd> + <kbd>enter</kbd>
+* [Open link with keyboard](#creating-a-link) Selector on text + <kbd>ctrl</kbd> + <kbd>enter</kbd>
+* [Open link with mouse](#creating-a-link) <kbd>Alt</kbd> + double-click on link
+* [Autocomplete find document](#creating-a-link) <kbd>[</kbd> + <kbd>[</kbd>
+* [Autocomplete to insert link](#using-auto-completion-to-insert-note-links) <kbd>ctrl</kbd> + <kbd>space</kbd>
+* [Find references](#searching-for-friends) Selector in link + <kbd>alt</kbd> + <kbd>enter</kbd>
+* [View all tags](#getting-an-overview-of-all-your-tags) <kbd>#</kbd> + <kbd>!</kbd>
+* [Autocomplete tag](#experimental-tag-selector) <kbd>#</kbd> + <kbd>?</kbd>
+* [Find tag references](#) Selector in tag <kbd>ctrl</kbd> + <kbd>enter</kbd>
+* [Expand link inline](#inline-expansion-of-note-links) <kbd>ctrl</kbd> + <kbd>.</kbd>
+* [Insert multimarkdown citation](#inserting-a-citation) <kbd>[</kbd> + <kbd>#</kbd>
```

Comment:
actually, the type of citation depends on `"citations-mmd-style": true,` (for multimarkdown) and `"citations-mmd-style": false,` (for pandoc = default).